Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ruby:2.3.1
+MAINTAINER Kevin Reedy <kreedy@chef.io>
+
+RUN apt-get update && apt-get install -y fakeroot
+
+ADD . /opt/chefdk-build
+
+WORKDIR /opt/chefdk-build/omnibus
+RUN bundle install --without development
+RUN git config --global user.email "nobody@chef.io" && git config --global user.name "Chef"
+RUN bundle exec omnibus build chefdk && bundle exec omnibus clean chefdk
```

Comment:
@tyler-ball: It'd be great to have this automatically built (and published to Docker Hub (that's my next step)) on every tagged release. I typically uninstall/reinstall chef-dk a ton to reproduce bugs.

As far as building vs installing the already-built omnibus package, it might be a chicken/egg problem depending on if the omnibus build gets triggered by the tag, or if it sets the tag at the end of the build. This would also allow someone to quickly test local changes before sending in a PR (at least for linux builds).

Building on tags in the official project also means nobody needs to keep [updating separate projects](https://hub.docker.com/search/?isAutomated=0&isOfficial=0&page=1&pullCount=0&q=chefdk+chef-dk&starCount=0)